packages/babel-plugin-proposal-logical-assignment-operators/src/index.js

```diff
@@ -39,11 +39,18 @@ export default declare(api => {
           }
         }
 
+        const isRHSAnonymousFunction = t.isFunctionExpression(right, {
```

Comment:
We should also do it for arrow functions:
```js
a ??= () => {};
```
shouldn't get an implicit name.

Reply:
Good catch! Let me fix it.